### PR TITLE
fix image building action for extract correct name from paths

### PR DESCRIPTION
**Pull request by ca7alindev**

### CODE REVIEW

**Review by web7evgheni — APPROVED** [review]

File: build.txt
Comment (web7evgheni) [inline build.txt:1]: @ca7alindev Please exclude this file

File: name
Comment (web7evgheni) [inline name:1]: @ca7alindev Please exclude this file

**Review by web7evgheni — CHANGES_REQUESTED** [review]

**Comment by web7evgheni** [comment]
@ca7alindev please remove files, i can't merge.

File: name
Comment (ca7alindev) [reply name:1]: image

**Review by web7evgheni — APPROVED** [review]

**Pull request merged**